blob/helper.go

```diff
@@ -0,0 +1,35 @@
+package blob
+
+import (
+	"github.com/celestiaorg/celestia-app/pkg/shares"
+)
+
+// sharesToBlobs takes raw shares and converts them to blobs.
+func sharesToBlobs(rawShares [][]byte) ([]*Blob, error) {
+	if len(rawShares) == 0 {
+		return nil, errBlobNotFound
+	}
+
+	shareSequences, err := shares.ParseShares(rawShares)
+	if err != nil {
+		return nil, err
+	}
+
+	blobs := make([]*Blob, len(shareSequences))
+	for i, sequence := range shareSequences {
+		data, err := sequence.RawData()
+		if err != nil {
+			return nil, err
+		}
+		if len(data) == 0 {
+			continue
```

Comment:
Yes, because `data, err := sequence.RawData()` returns a slice of the shares of the single blob. if it's empty, then we could not get shares for some reason. Added just as a sanity check, because we are using data[0] on the next line.

Reply:
[not blocking]
we might want to document that above